src/lotus_json/hash_map.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019-2024 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::*;
+use ahash::HashMap as AHashMap;
+use std::hash::Hash;
+
+impl<K, V> HasLotusJson for AHashMap<K, V>
```

Comment:
nonblocking:
```rust
K: HasLotusJson,
K::HasLotusJson: Eq + Hash
```

Also `TODO` that this suffers the same recursion issues as `impl LotusJson for Vec<T>`, but they should be fixed together.

I'm surprised that this doesn't need to `impl JsonSchema`

Reply:
`AHashMap` is a type alias of `StdHashMap`, which has `JsonSchema` implemented, see https://docs.rs/schemars/latest/schemars/trait.JsonSchema.html#impl-JsonSchema-for-HashMap%3CK,+V,+H%3E